pkg/feature/feature.go

```diff
@@ -75,12 +75,12 @@ func (f *Feature) Apply() error {
 	}
 
 	// Process and apply manifests
-	for _, m := range f.manifests {
-		if err := m.processTemplate(f.Spec); err != nil {
+	for i, m := range f.manifests {
+		if err := m.processTemplate(embeddedFiles, f.Spec); err != nil {
 			return errors.WithStack(err)
 		}
 
-		log.Info("converted template to manifest", "feature", f.Name, "path", m.targetPath())
```

Comment:
I think as a follow-up PR should change log levels from info to debug in most places. It's too much meaningless output in the logs now just to see that stuff is happening